drivers/virtio/net/virtio_net.c

```diff
@@ -893,12 +895,12 @@ static __u16 virtio_net_mtu_get(struct uk_netdev *n)
 	return d->mtu;
 }
 
-static int virtio_netdev_feature_negotiate(struct uk_netdev *n)
+static int virtio_netdev_probe(struct uk_netdev *n)
 {
-	__u64 host_features = 0;
-	__u64 drv_features  = 0;
-	int rc = 0;
 	struct virtio_net_device *vndev;
+	__u64 drv_features = 0;
+	__u64 host_features;
+	int rc = 0;
```

Comment:
Hmm, I see in case of success you `return 0;` regardless. So this early initialization is probably not needed.